includes/admin/feedzy-rss-feeds-import.php

```diff
@@ -3094,7 +3094,7 @@ public function add_import_actions( $actions, $post ) {
 				) : '#',
 				feedzy_is_pro() ? 'fz-export-btn' : 'fz-export-btn-pro',
 				! feedzy_is_pro() ? 'opacity:0.5;' : '',
-				( ! feedzy_is_pro() ? esc_html__( 'Export', 'feedzy-rss-feeds' ) . '<span style="font-size: 13px;line-height: 1.5em;width: 13px;height: 13px;" class="dashicons dashicons-lock"></span>' : '' )
+				( ! feedzy_is_pro() ? esc_html__( 'Export', 'feedzy-rss-feeds' ) . '<span style="font-size: 13px;line-height: 1.5em;width: 13px;height: 13px;" class="dashicons dashicons-lock"></span>' : esc_html__( 'Export', 'feedzy-rss-feeds' ) )
```

Comment:
This can be simplified like this:

```php
esc_html__( 'Export', 'feedzy-rss-feeds' ) . ( ! feedzy_is_pro() ?  '<span style="font-size: 13px;line-height: 1.5em;width: 13px;height: 13px;" class="dashicons dashicons-lock"></span>' : ''
```

Which is in a similar form to the original code before the change: https://github.com/Codeinwp/feedzy-rss-feeds/commit/d8b11faed800a4cc183fbff513967568cf4ee975